Sources/VelociPlayer/Source/VelociPlayer+Observation.swift

```diff
@@ -149,4 +149,11 @@ extension VelociPlayer {
             await self.updateCurrentItemDuration()
         }
     }
+    
+    internal func observePlayerOnUserInteraction() {
```

Comment:
Since we already have `startObservingPlayer()`, it might get confusing to have such similar names (even though this is an internal function). Could we come up with something that makes more sense? Maybe something along the lines of `beginPlayerObservationIfNeeded()`, or `attemptInitialPlayerObservation()`?

Reply:
Ok great! I opted in for beginPlayerObservationIfNeeded, I think this name makes more sense.